packages/atomic/src/components.d.ts

```diff
@@ -1081,6 +1113,42 @@ export namespace Components {
          */
         "label": string;
     }
+    interface AtomicSvcInsightInterface {
```

Comment:
Why is it called svc?

Reply:
For `service`. I did a poll in a shared channel with other teams, and this was what seemed to reach a consensus.

Reply:
Service